packages/web-frontend/src/containers/MapOverlayBar/Content.js

```diff
@@ -6,14 +6,15 @@ export const Content = styled.div`
   justify-content: space-between;
   align-items: flex-start;
   pointer-events: auto;
-  margin: 21px 16px 20px 16px;
+  padding: 21px 16px 15px 16px;
   color: #ffffff;
 `;
 
 export const ContentText = styled.div`
   font-weight: 500;
   font-size: 16px;
-  line-height: 19px;
+  line-height: 21px;
+  padding-right: 15px;
```

Comment:
As discussed with David, we want to add a padding between the overlay name and switch button, see screenshot: https://linear.app/bes/issue/PHX-130#comment-3097748d